examples/custom_components_playground/src/main.rs

```diff
@@ -0,0 +1,5 @@
+pub mod tests;
+
+fn main() {
+    tests::jazda();
```

Comment:
jak mu to mamy pokazywać to może zmienic nazwe można

Reply:
nie